bb8/src/api.rs

```diff
@@ -74,6 +74,14 @@ impl<M: ManageConnection> Pool<M> {
     }
 }
 
+#[derive(Debug)]
+pub enum QueueStrategy {
+    /// Last in first out
+    LIFO,
```

Comment:
We should spell the variant name `Lifo` instead of `LIFO` per the naming guidelines. IMO the comment in its current state isn't that useful, I think it would be helpful to elaborate on the intended effect.

Also, please move the type definition below the `Builder` impl -- the prevailing style is sort of top-down.

I think it would be useful to derive `Default` on `QueueStrategy`, and then initialize `Builder::queue_strategy` to `QueueStrategy::default()`.

Reply:
Thanks for the suggestions, I've implemented them in my latest changes but still failing the build tests. Can't reproduce on my machine so let me know if you have any ideas on how to pass them

Reply:
That's right -- I said to derive `Default` but that's not supported yet by the MSRV. Can you just add a custom `Default` `impl` that does the same thing instead? I like to avoid bumping the MSRV where possible.